app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -21,29 +21,116 @@
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_margin="16dp"
+                        android:layout_marginLeft="@dimen/spacing_normal"
+                        android:layout_marginTop="@dimen/activity_vertical_margin"
+                        android:layout_marginRight="@dimen/spacing_normal"
+                        android:adjustViewBounds="true"
+                        android:background="?android:selectableItemBackground"
+                        android:maxHeight="120dp"
+                        android:scaleType="fitCenter"
+                        android:transitionName="@string/product_transition"
+                        app:srcCompat="@drawable/ic_add_a_photo_black_48dp" />
+
+                    <TextView
+                        android:id="@+id/addPhotoLabel"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="center"
+                        android:text="@string/take_picture_front"
+                        android:textStyle="bold" />
+
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    android:layout_toRightOf="@id/front_picture_layout"
+                    android:layout_centerVertical="true"
+                    android:id="@+id/nutrition_image_prompt_layout"
+                    android:visibility="gone">
+
+                    <ImageButton
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_margin="16dp"
+                        android:layout_marginLeft="@dimen/spacing_normal"
+                        android:layout_marginTop="@dimen/activity_vertical_margin"
+                        android:layout_marginRight="@dimen/spacing_normal"
+                        android:adjustViewBounds="true"
+                        android:background="?android:selectableItemBackground"
+                        android:maxHeight="120dp"
+                        android:scaleType="fitCenter"
+                        android:transitionName="@string/product_transition"
+                        app:srcCompat="@drawable/ic_add_a_photo_black_48dp"
+                        android:id="@+id/imageViewNutrition"/>
+
+                    <TextView
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="center"
+                        android:text="Take a picture of the nutrient facts"
```

Comment:
You must extract the strings .

Reply:
Take a picture of the nutrition facts

Reply:
The strings have been extracted as resources. Kindly review the latest commit.